core-bundle/src/Resources/contao/classes/Ajax.php

```diff
@@ -158,7 +158,9 @@ public function executePreActions()
 	 */
 	public function executePostActions(DataContainer $dc)
 	{
-		header('Content-Type: text/html; charset=' . System::getContainer()->getParameter('kernel.charset'));
+		$response = (new Response('', 200, array('content-type' => 'text/html')))
+			->setCharset(System::getContainer()->getParameter('kernel.charset'))
+		;
```

Comment:
this is unnecessary because it is already the default for all responses. I think we can just drop this line and not change anything else...